apps/rig/lib/rig/application.ex

```diff
@@ -2,10 +2,15 @@ defmodule Rig.Application do
   @moduledoc false
 
   use Application
+  use Rig.Config, []
```

Comment:
why not require it?
```elixir
use Rig.Config, [:log_level]
```

Reply:
was experimenting with something and didn't put it back